pages/common/touch.md

```diff
@@ -14,3 +14,7 @@
 - Use the times from a file to set the times on a second file:
 
 `touch -r {{filename}} {{filename2}}`
+
+- Create multiple files:
+
+`touch app{1,2,3}.txt`
```

Comment:
I'm not sure about this example.  While definitely useful, it seems to demonstrate typical shell functionality (i.e. glob patterns) rather than behavior specific to the `touch` utility itself.  I'll let others way in here, ping @sbrl.

Reply:
I like the example, find my self using -d for ls often, just make it ls -d */ then.

Reply:
Yeah, I think the `ls -d */` is a great one.  But I'm not so sure about this `touch app{1,2,3}.txt` one.  Let's wait and see what others think.

Reply:
@bl-ue Since the touch page does not reach the limitation, how about adding this one. I think this example is useful sometimes.

Reply:
Looks like shell-specific to me. Generally we try to document the features of a command, rather than the shell it's running in.